deployments/docker/docker-compose.yml

```diff
@@ -0,0 +1,93 @@
+version: '3.8'
+
+services:
+  provisioning:
+    build:
+      context: ../../
+      dockerfile: ./deployments/docker/Dockerfile
+      target: provisioning-run
+      args:
+        PROVISIONING_DEPLOY_PREFIX: ${PROVISIONING_DEPLOY_PREFIX:?Please define a vlaue for the environment variable}

```

Comment:
I thought the error statement needed quoting.  (I'm very likely to be wrong.)

Reply:
If I add quotes it also outputs the quotes like this, but it can exist without quoting:
`required variable PROVISIONING_DEPLOY_PREFIX is missing a value: 'Please define a value for the environment variable'`

Which formatting would you prefer?

Reply:
It's great as it is now.  Sorry for the noise.